### PR TITLE
Added back to top

### LINKED ISSUE #1334: Add back to top button
**Is your feature request related to a problem? Please describe.**
NA

**Describe the solution you'd like**
NA

**Describe alternatives you've considered**
NA

**Additional context**
I want to add back to top button. please assign me. I'll do it asap.

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by charandeep7**
## Purpose
I've added a back to top button. It'll help user to reach to the top of page more easily.
The purpose of this PR is to fix #1334 

## Goals
Users can reach to the top of page in a single click.

### Screenshots
BackToTop

  
### Preview Link

https://pr-1335-sef-site.surge.sh/

##  Checklist
- [X] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [X] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by YoshithaRathnayake** [comment]
It looks good if it is little bit small in mobile view right? @anjula-sack ?

File: assets/js/back2top.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    btn.addClass('show');
+  } else {
+    btn.removeClass('show');
+  }
+});
+
+btn.on('click', function(e) {
+  e.preventDefault();
+  $('html, body').animate({scrollTop:0}, '300');
+});
```
Comment (anjula-sack) [inline assets/js/back2top.js:13]: Can you add the end of the line?

File: index.html [hunk 2]
```diff
@@ -506,6 +547,7 @@ <h5 class="name">Sanmugathasan Prasanthan</h5>
     <p class="small text-muted text-left mb-0"> *Our mentors and speakers engage in mentoring and speaking voluntarily
         in their leisure time<br> and we don't have a direct affiliation with the mentioned organisations</p>
 </script>
+<script src="assets/js/back2top.js"></script>
```
Comment (anjula-sack) [inline index.html:550]: Shall we put the js script here without an external file?

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Thank you! @charandeep7 Small changes requested

**Comment by anjula-sack** [comment]
> It looks good if it is little bit small in mobile view right? @anjula-sack ?

yeah maybe we can change the color as well.

**Comment by YoshithaRathnayake** [comment]
> > It looks good if it is little bit small in mobile view right? @anjula-sack ?
> 
> yeah maybe we can change the color as well.

Yeah!

**Comment by Madhawa97** [comment]
I think a circle shape with a thinner arrow like the following image and would be suited for the site. 
Screenshot 2022-10-22 at 20 02 47

**Comment by YoshithaRathnayake** [comment]
> I think a circle shape with a thinner arrow like the following image and would be suited for the site. Screenshot 2022-10-22 at 20 02 47

Yes that's nice

File: index.html (hunk 2 above)
Comment (jayasanka-sack) [reply index.html:550]: Having an external file would help us to include it on other pages as well.

Comment (jayasanka-sack) [reply index.html:550]: In that case, css should be also moved to a separate file.

File: assets/js/back2top.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+var btn = $('#backToTop');
```
Comment (jayasanka-sack) [inline assets/js/back2top.js:1]: Let's avoid using a global variable here. 

@RandilaP you can explain why.

**Comment by jayasanka-sack** [comment]
Thanks for your great PR @charandeep7! ❤️ 

However, I'm not clear why we need one. My humble opinion is it won't add much value to the user and might slightly decrease the professional value of the site. We need to pay extra attention when adding fixed elements to the screen, especially on mobile as we have limited space. 

The only use case I can think of is to save time for users on reaching the scroll bar to scroll up. However, I don't think people manually click the scrollbar to scroll anymore as nowadays devices come with gestures to scroll. ex: two-finger scroll on touchpads, the scroll wheel on the mouse and mobile devices do not need the scrollbar at all. 

Therefore I think we can close this PR. Thanks again for your kind contribution, it means to us a lot! 🤗❤️ We will keep `hacktoberfest-accepted` badge for this PR so that it will still count in your hacktoberfest dashboard. 

@anjula-sack @Piumal1999 @YoshithaRathnayake @Madhawa97 let me know your thoughts.

**Comment by Madhawa97** [comment]
Yeah @jayasanka-sack. In mobile view it feels a bit crowded. May be we add this to the `success stories page` only. SInce its quite a lengthy page? I'm not sure whether if it's weird to have this only on a single page though.

**Comment by YoshithaRathnayake** [comment]
> Thanks for your great PR @charandeep7! ❤️
> 
> However, I'm not clear why we need one. My humble opinion is it won't add much value to the user and might slightly decrease the professional value of the site. We need to pay extra attention when adding fixed elements to the screen, especially on mobile as we have limited space.
> 
> The only use case I can think of is to save time for users on reaching the scroll bar to scroll up. However, I don't think people manually click the scrollbar to scroll anymore as nowadays devices come with gestures to scroll. ex: two-finger scroll on touchpads, the scroll wheel on the mouse and mobile devices do not need the scrollbar at all.
> 
> Therefore I think we can close this PR. Thanks again for your kind contribution, it means to us a lot! 🤗❤️ We will keep `hacktoberfest-accepted` badge for this PR so that it will still count in your hacktoberfest dashboard.
> 
> @anjula-sack @Piumal1999 @YoshithaRathnayake @Madhawa97 let me know your thoughts.

Is it bad if we add it only for the desktop view @jayasanka-sack

File: assets/js/back2top.js (hunk 3 above)
Comment (YoshithaRathnayake) [reply assets/js/back2top.js:1]: So @jayasanka-sack we can use there a 'let' without the 'var' right?

**Comment by jayasanka-sack** [comment]
Let's close this PR. Thanks @charandeep7 !

**Pull request closed without merging**